lib/workers/repository/update/pr/index.ts

```diff
@@ -57,6 +64,37 @@ export type ResultWithoutPr = {
 
 export type EnsurePrResult = ResultWithPr | ResultWithoutPr;
 
+export function updatePrRenovateVerData(
+  existingPr: Pr | undefined,
+  prBody: string
+): string {
+  let res = prBody;
+  let oldData;
+  let prDataNew: RenovatePrData;
+  if (existingPr) {
+    oldData = existingPr.bodyStruct?.renovatePrVerData;
+    if (oldData) {
+      prDataNew = JSON.parse(oldData);
+      prDataNew.prUpdateVer = pkg.version;
+    } else {
+      prDataNew = { prCreationVer: 'unknown', prUpdateVer: pkg.version };
```

Comment:
please suggest what should be the value instead of unknown.

Reply:
@rarkins this is the case where we didn't find any `prData` in the current PR. i set here
prDataNew = { prCreationVer: '', prUpdateVer: pkg.version };

should be ok right?